src/components/pomodoro-pane.js

```diff
@@ -1,9 +1,16 @@
 import { useEffect, useState } from "react";
+import { useTimer } from "../context/timer-context";
 
 const PomodoroPane = () => {
-  const [timer, setTimer] = useState(1500);
-  const [pause, setPause] = useState(true);
-  const [menuOptionActive, setMenuOptionActive] = useState("Pomodoro");
+  const { timerState, timerDispatch } = useTimer();
+  const {
+    pause,
```

Comment:
You can make this state locally where you are using this.

Reply:
done, thanks for the suggestion